KdybyCodingStandard/ruleset-7.1.xml

```diff
@@ -48,6 +48,11 @@
         <exclude name="SlevomatCodingStandard.TypeHints.DisallowArrayTypeHintSyntax.DisallowedArrayTypeHintSyntax"/>
         <exclude name="SlevomatCodingStandard.TypeHints.DisallowMixedTypeHint.DisallowedMixedTypeHint"/>
         <exclude name="SlevomatCodingStandard.TypeHints.TypeHintDeclaration.MissingTraversablePropertyTypeHintSpecification"/>
+        <exclude name="SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly"/>
```

Comment:
tohle zní užitečně, pro jsi to dropnul?

Reply:
Dělalo to že každou globální funkci dal fixer do use což je v konfliktu s jiným už includovanym pravidlem které přidává takovým metodám lomítko na začátek. A taky jsem tuhle podivnost ještě nikde neviděl používat 😁.